numpy/core/src/umath/loops_arithm_fp.dispatch.c.src

```diff
@@ -22,21 +22,25 @@
  ********************************************************************************/
```

Comment:
You should create new dispatch-able source for integer arithmetic operations, this source marked for float points.
The new dispatch-able source should target `SSE41` and `AVX512_SKX` -> (`NPY_HAVE_AVX512VL NPY_HAVE_AVX512BW NPY_HAVE_AVX512DQ`) according to the nature of eumlating.

> loops_arithmetic.dispatch.c.src
```C
/*@targets
 ** $maxopt baseline
 ** sse2 sse41 avx2 avx512_skx
 ** vsx2
 ** neon 
 **/

#include "simd/simd.h"
#include "loops.h"

#if NPY_SIMD // your NPYV kernels

#endif
// or raw x86 SIMD
#if defined(NPY_HAVE_AVX512BW)

#elif defined(NPYV_HAVE_AVX2)

#elif defined(NPYV_HAVE_SSE41)

#elif defined(NPYV_HAVE_SSE2)

#endif
// or raw VSX
#ifdef NPY_HAVE_VSX2

#endif
// or raw NEON
#ifdef NPY_HAVE_NEON

#endif


/**begin repeat
 * Unsigned types
 *  #type = npy_ubyte, npy_ushort, npy_uint, npy_ulong, npy_ulonglong#
 *  #TYPE = UBYTE, USHORT, UINT, ULONG, ULONGLONG#
 */
NPY_NO_EXPORT void NPY_CPU_DISPATCH_CURFX(@TYPE@_divide)
(char **args, npy_intp const *dimensions, npy_intp const *steps, void *NPY_UNUSED(func))
{
}
/**end repeat**/ 
```

once you add the new file to the source tree, you will still need to add another two steps:

- forward declarations of ufunc targets, for example:
  https://github.com/numpy/numpy/blob/f5f845b7bd0b737740e419826698ab4a9e61564d/numpy/core/src/umath/loops.h.src#L194-L202

- runtime dispatching call for each ufunc, for example:
  https://github.com/numpy/numpy/blob/f5f845b7bd0b737740e419826698ab4a9e61564d/numpy/core/code_generators/generate_umath.py#L339

Reply:
Ohh ok, I totally missed `fp` in loops_arithm_fp :). Ok I shall try that thanks.